sigstore/rekor/_client.py

```diff
@@ -0,0 +1,7 @@
+"""
+Client implementation for interacting with Rekor.
+"""
+
+
+class RekorClient:
```

Comment:
Rekor's API seems to be a lot more complicated. The Rekor client in Go is generated via Swagger [here](https://github.com/sigstore/rekor/tree/main/pkg/generated/client).

We might have to figure out the main things we want/need to support for the beta.